### PR TITLE
update README.md with JupyterLite site instructions

### PR DESCRIPTION
**Pull request by Vipul-Cariappa**
Look at https://github.com/jupyter-xeus/xeus/issues/377#issuecomment-1883041231

### CODE REVIEW

**Comment by martinRenou** [comment]
Thanks! @Vipul-Cariappa 

It seems you're not getting credits for your changes in Github UI. You are creating your commits with an email address ending with `@.com` (see https://github.com/jupyterlite/xeus/commit/078cc6f641bb241bc9953fef7b5e5b904af4603a.patch). 

It seems it's not a valid email address? This may be fine if you add this address to your Github account https://github.com/settings/emails (no need to verify your email address for Github to consider the commits coming from this address to be yours). Though you may want to fix this in your git config.

**Comment by Vipul-Cariappa** [comment]
@martinRenou, Thanks for pointing out that my email configured is wrong. I have fixed it using `git config --global user.email "vipulcariappa@gmail.com"`.
Any commits in the future should work fine.

**Comment by Vipul-Cariappa** [comment]
Hope the failing test is not my doing?

File: README.md [hunk 1]
```diff
@@ -125,6 +125,22 @@ emcmake cmake \
...
+```bash
+# create new environment
+micromamba create -n xeus-lite-host \
+    jupyterlite-core
+
+# activate the environment
+micromamba activate xeus-lite-host
+
+# install jupyterlite_xeus via pip
+python -m pip install jupyterlite_xeus
```
Comment (martinRenou) [inline README.md:141]:
```suggestion
python -m pip install jupyterlite-xeus
```

**Review by martinRenou — COMMENTED** [review]
Just a tiny comment, otherwise looks good thanks!

The CI failure seems to be a flaky test, not yours.

**Pull request merged**